build.sbt

```diff
@@ -1,20 +1,28 @@
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
+lazy val root = (project in file("."))
+  .settings(
```

Comment:
If you update to sbt `1.2.3`, you need to add here `  .enablePlugins(SbtPlugin)`, see https://www.scala-sbt.org/1.0/docs/Testing-sbt-plugins.html#step+2%3A+SbtPlugin